docs/implementation/metadata.md

```diff
@@ -141,3 +141,16 @@ correctly across index node restarts.
 
 The table `subgraphs.table_stats` stores which tables for a deployment
 should have the 'account-like' optimization turned on.
+
+### `subgraphs.subgraph_features`
+
+Details about features that a deployment uses
```

Comment:
This should indicate that that table is stored in the primary (I think that's where it lives, right?)

Reply:
Indicated that.